nemo/collections/asr/parts/numba/rnnt_loss/rnnt.py

```diff
@@ -180,7 +180,7 @@ def rnnt_loss_gpu(
     acts, acts_shape = rnnt_helper.flatten_tensor(acts)
 
     ### REPRESENT THE CUDA ARRAY INTERFACE OF COSTS VECTOR ###
-    costs_repr = cuda.as_cuda_array(costs)  # NO COPY OF DATA, JUST CHANGE REPRESENTATION
+    costs_repr = cuda.as_cuda_array(costs, sync=False)  # NO COPY OF DATA, JUST CHANGE REPRESENTATION
```

Comment:
Is it OK to not sync it?

Reply:
Yup tested this. Was the cause of a minor slow down since it would wait for the default pytorch stream to execute and sync before continuing, which was unnecessary since these kernels will just queue more stuff to that same stream.